docs/experimental-windows.md

```diff
@@ -49,33 +47,37 @@ metadata:
 spec:
   strictAffinity: true
 ```
-Another way is to use calicoctl manually:
+Alternately, you can manually execute calicoctl:
 ```
 calicoctl ipam configure --strictaffinity=true
 ```
 
 ### Network connectivity in AWS
-The network interface attached to your EC2 instance MUST have disabled “Change Source/Dest. Check” option.
-In AWS console option can be found on the Actions menu for a selected network interface.
+
+Disable the `Change Source/Dest. Check` option for the network interface
+attached to your EC2 instance. In AWS, the console option for the network
+interface is in the **Actions** menu.
 
 ### Hacks
 
-We need to figure out proper way to pass cluster settings from controller plane to worker.
+k0s offers the following CLI arguments in lieu of a formal means for passing
+cluster settings from controller plane to worker:
 
-While we don't have it, there are CLI arguments:
 - cidr-range
 - cluster-dns
 - api-server 
 
 
-## Some useful commands
+## Useful commands
+
+### Run pod with cmd.exe shell###
```

Comment:
Something happened with the `#`s here

Reply:
Yes. No idea what. The extras have all been sent home now, though. :-)